tsconfig_triggers.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Does it make sense to do this programmatically instead (feel free to say no)?
https://github.com/Microsoft/TypeScript/wiki/Using-the-Compiler-API#a-minimal-compiler

Reply:
Oh yeah, that'd be a much better way to do it.

Reply:
Also a much more complicated way.  I'll uh leave a comment for future me.